### PR TITLE
TECH-1079 Bug: Fix validations in update order status

### PR DESCRIPTION
**Pull request by David25848**
**Refer:** 

- https://app.clickup.com/t/3077149/TECH-1079 
- https://app.clickup.com/t/3077149/TECH-631 
- https://shipitchile.atlassian.net/browse/SD-1025

**Why I'm ading these lines of code:**

- Validate shipit statuses send to update woocommerce orders
- Avoid create shipments when the shopping retired method is active
- Fix shipment of measurements for fulfillment clients
- Change 99minutos logo

**Test:**

Captura de pantalla de 2021-06-10 17-15-20
Captura de pantalla de 2021-06-10 17-16-25
Captura de pantalla de 2021-06-10 17-18-12
Screenshot from 2021-06-09 23-18-56

### CODE REVIEW

File: shipit/trunk/shipit.php [hunk 1]
```diff
@@ -607,9 +607,14 @@ function createShipment($company, $skus, $sellerSetting, $insuranceSetting, $ord
       if ($sellerSetting->automatic_delivery === false) {
         $integration = new Integration(get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token']);
         return $integration->massiveOrders(['orders' => [$orderPayload->build()]]);
-      } else {
-        $core = new Core(get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token'], 'v4');
-        return $core->shipments(['shipment' => $orderPayload->build()]);
+      } 
+      elseif($shipit) {
```
Comment (hiro-riveros) [inline shipit/trunk/shipit.php:611]:
```suggestion
      } elseif ($shipit)  {
```

File: shipit/trunk/shipit.php [hunk 2]
```diff
@@ -607,9 +607,14 @@ function createShipment($company, $skus, $sellerSetting, $insuranceSetting, $ord
...
+          $order = new WC_Order($order->get_id());
+          $order->add_order_note('Ya que aplicaron el metodo de retiro en tienda.');
+      }
+      else{
```
Comment (hiro-riveros) [inline shipit/trunk/shipit.php:615]:
```suggestion
      } else {
```

File: shipit/trunk/src/webhook.php [hunk 3]
```diff
@@ -259,8 +467,8 @@ function shipit_action_woocommerce_update_order(WP_REST_Request $request) {
     $int = (int)preg_replace('/\D/ui','',$json->reference);
     $order = new WC_Order($int);
     $statuses = wc_get_order_statuses();
-    $status_label = isset($statuses['wc-'.$status]) ? $status : 'other';
-    $result = $order->update_status($status_label, 'Estado actualizado por Shipit');
+    $status_label = isset($statuses['wc-'.$status]);
+    if($status_label) $order->update_status($status, 'Estado actualizado por Shipit');
```
Comment (hiro-riveros) [inline shipit/trunk/src/webhook.php:471]:
```suggestion
    if (isset($statuses['wc-'.$status])) $order->update_status($status, 'Estado actualizado por Shipit');
```

Comment (David25848) [reply shipit/trunk/src/webhook.php]: Thanks, All done

File: shipit/trunk/shipit.php (hunk 2 above)
Comment (David25848) [reply shipit/trunk/shipit.php]: done!

File: shipit/trunk/shipit.php (hunk 1 above)
Comment (David25848) [reply shipit/trunk/shipit.php]: done.

File: shipit/trunk/src/webhook.php (hunk 3 above)
Comment (David25848) [reply shipit/trunk/src/webhook.php]: @hiro-riveros

**Review by FelipeChibey — APPROVED** [review]

**Review by mgraterol — APPROVED** [review]

**Review by Barbycris — APPROVED** [review]

**Review by multinauta — APPROVED** [review]

**Review by gcamposm — APPROVED** [review]

**Review by luispascuali — APPROVED** [review]

**Review by FelipeChibey — APPROVED** [review]

**Review by multinauta — APPROVED** [review]

**Review by hiro-riveros — APPROVED** [review]

**Pull request merged**